### PR TITLE
[paypal][prohosted]	add purchase exemple

### PR DESCRIPTION
**Pull request by thiphamyp**
PR: https://github.com/Payum/Payum/pull/536

### CODE REVIEW

File: src/Acme/PaymentBundle/Controller/PaypalProHostedController.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        if ($form->isValid()) {
+            $data = $form->getData();
+
+            $storage = $this->getPayum()->getStorage(PaymentDetails::class);
+
+            /** @var $payment PaymentDetails */
+            $payment                  = $storage->create();
+            $payment['currency_code'] = $data['currency'];
+            $payment['subtotal']      = $data['amount'];
+            $payment['bn']            = 'FR_Test_H3S';
```
Comment (makasim) [inline src/Acme/PaymentBundle/Controller/PaypalProHostedController.php]: what is this?

Comment (thiphamyp) [inline src/Acme/PaymentBundle/Controller/PaypalProHostedController.php]: It's the partner code. It's not required but recommended for a partner Paypal

bn: Identifies the source that built the code for the button. 
(Ref: Page 19 of document: https://www.paypalobjects.com/webstatic/en_GB/developer/docs/pdf/hostedsolution_uk.pdf)

Comment (makasim) [inline src/Acme/PaymentBundle/Controller/PaypalProHostedController.php]: Could we remove it?

Comment (thiphamyp) [inline src/Acme/PaymentBundle/Controller/PaypalProHostedController.php]: I removed it

**Pull request merged**